routing/index_graph_starter.hpp

```diff
@@ -69,15 +73,25 @@ class IndexGraphStarter final
                                                 GetPoint(to, true /* front */));
   }
 
-  static bool IsFakeSegment(Segment const & segment) { return segment.GetFeatureId() == kFakeFeatureId; }
+  double CalcSegmentWeight(Segment const & segment) const
+  {
+    return m_graph.GetEstimator().CalcSegmentWeight(
+        segment, m_graph.GetRoadGeometry(segment.GetMwmId(), segment.GetFeatureId()));
+  }
+
+  static bool IsFakeSegment(Segment const & segment)
+  {
+    return segment.GetFeatureId() == kFakeFeatureId;
+  }
 
 private:
   static uint32_t constexpr kFakeFeatureId = numeric_limits<uint32_t>::max();
   static uint32_t constexpr kFakeSegmentIdx = numeric_limits<uint32_t>::max();
+  static NumMwmId constexpr kFakeMwmId = numeric_limits<NumMwmId>::max();
```

Comment:
Is it possible to create a named constant, say, kInvalidNumMwmId in num_mwm_id.hpp?